### PR TITLE
[Bug fix] LPS-84865

### PR DESCRIPTION
**Pull request by dantewang**
Sub task of bug https://issues.liferay.com/browse/LPS-84801

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

File: modules/apps/mobile-device-rules/mobile-device-rules-test/src/testIntegration/java/com/liferay/mobile/device/rules/service/test/MDRRuleGroupLocalServiceTest.java [hunk 1]
```diff
@@ -65,6 +75,111 @@ public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 	}
 
+	@Test
+	public void testCopyRuleGroup() throws Exception {
```
Comment (dantewang) [inline modules/apps/mobile-device-rules/mobile-device-rules-test/src/testIntegration/java/com/liferay/mobile/device/rules/service/test/MDRRuleGroupLocalServiceTest.java:79]: @yuhai Could you confirm that the integration test can expose the issue before fixing? That means the integration test should fail if the local service doesn't copy the permissions.

Comment (yuhai) [reply modules/apps/mobile-device-rules/mobile-device-rules-test/src/testIntegration/java/com/liferay/mobile/device/rules/service/test/MDRRuleGroupLocalServiceTest.java:79]: @dantewang, testCopyRuleGroup method can't expose the issue, it tested copy default permissions. testCopyRuleGroupWithoutOwnerViewPermission method can expose the issue before fixing.

Comment (dantewang) [reply modules/apps/mobile-device-rules/mobile-device-rules-test/src/testIntegration/java/com/liferay/mobile/device/rules/service/test/MDRRuleGroupLocalServiceTest.java:79]: @yuhai Could you add the integration test to expose the issue first, and then fix it? Also, I think it's OK to just test copying the MDRRuleGroup with modified permissions -- compare every permission to make sure all of them are equal (copied).

Comment (yuhai) [reply modules/apps/mobile-device-rules/mobile-device-rules-test/src/testIntegration/java/com/liferay/mobile/device/rules/service/test/MDRRuleGroupLocalServiceTest.java:79]: @dantewang, ok, I will do this.

**Comment by yuhai** [comment]
@dantewang, resend it, please see https://github.com/liferay-core-infra/liferay-portal/pull/13

**Comment by yuhai** [comment]
ci:close

**Pull request closed without merging**